### DISCUSSION TITLE
Directus BE not working after domain change

### DISCUSSION THREAD

**Discussion by andriusign** [directus/directus#5373] [Archive]
Directus version: directus:v9.0.0-rc.51
Running in Docker

After migrating from domaina.com to domainb.com BE admin is not working and responds with messages in console:
![image](https://user-images.githubusercontent.com/14263959/116601330-0e1add80-a933-11eb-902f-ce0360ffb6af.png)

_data_ and _uploads_ folders have been migrated to new server.
In database _directus_setttings_ table project url has been set to new domain. 

Interestingly front-end app is still accessing BE and loading items, letting login etc.

**Comment by rijkvanzanten** [comment]
Is the public_url environment variable set correctly? Is your server hosted on `/api` correctly? How are you doing the routing from `/api` to the Directus instance? Is the URL in the warning below the same as what you're trying to access?
